entity-framework/core/querying/filters.md

```diff
@@ -25,20 +25,20 @@ First, define the entities:
 
 [!code-csharp[Main](../../../samples/core/QueryFilters/Program.cs#Entities)]
 
-Note the declaration of a __tenantId_ field on the _Blog_ entity. This will be used to associate each Blog instance with a specific tenant. Also defined is an _IsDeleted_ property on the _Post_ entity type. This is used to keep track of whether a _Post_ instance has been "soft-deleted". That is, the instance is marked as deleted without physically removing the underlying data.
```

Comment:
Isn't "_tendantId" correct here?

Reply:
before my change it had two `_` which resulted in the entire line being bold